Makefile

```diff
@@ -76,18 +76,18 @@ refresh-certs: clean
 install-bundle:
 	$(MKDIR) $(DESTDIR)
 	$(CP) $(BUNDLE) $(DESTDIR)
-	$(CHOWN) $(USER):$(GROUP) $(DESTDIR)/$(BUNDLE)
-	$(CHMOD) $(PERMISSIONS) $(DESTDIR)/$(BUNDLE)
+	sudo $(CHOWN) $(USER):$(GROUP) $(DESTDIR)/$(BUNDLE)
```

Comment:
In many containers where we build this, we run as root and there is no `sudo` available. We probably need to detect if sudo exists and omit it if not.  Probably do `SUDO := $(shell which sudo)` above and then use `$(SUDO)` here so it's just blank if sudo doesn't exist and it won't fail.

Reply:
Sounds good, commit added and pushed up.